sample/AirshipSample/android/build.gradle

```diff
@@ -22,18 +23,12 @@ buildscript {
 
 allprojects {
     repositories {
-        google()
```

Comment:
this ordering is important, was this on purpose?

Reply:
mavenLocal() normally is first, so its fine. Just needs to be ahead of jcenter()